be/src/column/column_pool.h

```diff
@@ -135,8 +135,8 @@ class CACHELINE_ALIGNED ColumnPool {
             return obj;
         }
 
-        inline void return_object(T* ptr) {
-            if (UNLIKELY(column_reserved_size(ptr) > config::vector_chunk_size)) {
+        inline void return_object(T* ptr, size_t batch_size) {
+            if (UNLIKELY(column_reserved_size(ptr) > batch_size)) {
```

Comment:
In order to prevent ColumnPool from occupying a large block of memory and not releasing it, those that exceed ChunkSize will not enter ColumnPool, so that this semantics will be lost after the change.